control/modelsimp.py

```diff
@@ -211,26 +218,46 @@ def balred(sys, orders, method='truncate'):
         of systems)
     method: string
         Method of removing states, either ``'truncate'`` or ``'matchdc'``.
+    alpha: float
+        Specifies the alpha-stability boundary for the eigenvalues
```

Comment:
This text description of `alpha` appears to be copied from AB09MD.f of the SLICOT source code. Is that true? SLICOT is under the GNU GPL, whereas we want to keep this `control` package under [the BSD license](https://github.com/python-control/python-control/blob/c1f5a9e06fa1cce06a859edcfb41b257ab772372/LICENSE).